src/main/java/org/prebid/server/auction/LatestRatesService.java

```diff
@@ -0,0 +1,50 @@
+package org.prebid.server.auction;
+
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.json.Json;
+import org.prebid.server.auction.model.Currency;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.Map;
+
+/**
+ * Class holding and updating latest currencies from server.
+ */
+public class LatestRatesService {
+
+    private static final String CURRENCY_SERVER_URL = "currency.prebid.org";
+    private static final Logger logger = LoggerFactory.getLogger(LatestRatesService.class);
+
+    private HttpClient httpClient;
+    private volatile Currency currency;
+
+    public LatestRatesService(HttpClient httpClient) {
+        this.httpClient = httpClient;
+    }
+
+    /**
+     * Updates rate from latest currency server.
+     */
+    public void updateRates() {
+        httpClient.getNow(80, CURRENCY_SERVER_URL, "/latest.json",
```

Comment:
`getAbs` would be more convenient here since it allows passing complete URL as a single parameter.

Also it would make the code more readable if handlers were extracted to separate methods.

Exception handling is desirable also at least for the sake of logging (and therefore visibility).